onboarding.md

```diff
@@ -0,0 +1,43 @@
+# Onboarding
+
+This document covers steps needed to onboard new Steering Committee
+members and off-board emeritus members.
+
+- [ ] All new members must complete the [Inclusive Leadership Training].
+
+- [ ] Update the following files in the respective repos:
+  - [ ] [kubernetes/steering]: `OWNERS_ALIASES` and `README.md`
+  - [ ] [kubernetes/community]: `sigs.yaml` and auto-generated content
+  - [ ] [kubernetes/org]: `OWNERS_ALIASES` and the `steering-committee`
+    GitHub team
+  - [ ] [kubernetes/funding]: `OWNERS_ALIASES`
+  - [ ] [kubernetes/kubernetes-template-project]: `OWNERS_ALIASES`
+
+- [ ] Add new members with Owner access to the `steering-private`
+  mailing list and remove emeritus members.
+
+- [ ] Grant new members Owner access to the public steering mailing
+  list and revoke access from emeritus members.
+
+- [ ] Add new members to the `steering-private` slack channel and
+  remove emeritus members.
+
+- [ ] Add new members to the CNCF Service Desk and remove emeritus
+  members.
+
+- [ ] Add new members to the [cncf-kubernetes-maintainers] mailing list.
```

Comment:
Do we want to remove new members too? I got added to the list recently, but from a quick glance it doesn't look like the list includes any confidential info...

Reply:
i don't mind if they want to be around. you don't really need to be on steering to get on this list AFAIK.